.babelrc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["@babel/preset-typescript"]
```

Comment:
This is a fine babel config but where does babel get invoked?

Reply:
I'm not sure what you mean by invoking babel

Reply:
We spoke offline, this is used by Jest but never invoked directly during the build.